### DISCUSSION TITLE
[System Property - ApplicationWorkbenchAdvisor] - "setAllViewsClosable=null"

### DISCUSSION THREAD

**Discussion by stvzldn** [mytourbook/mytourbook#1283] [General]
Is this an issue?  I have these info messages filling up my Error Log occurring every app start.

> "eclipse.buildId=unknown
> java.version=17.0.8
> java.vendor=Eclipse Adoptium
> BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
> Command-line arguments:  -os win32 -ws win32 -arch x86_64
> 
> net.tourbook.common
> Info
> Fri Feb 02 18:55:42 EST 2024
> 18:55:42.671'514 [System Property - ApplicationWorkbenchAdvisor] - "setAllViewsClosable=null" -> When "false" then all views will not be set closeable="true" in workbench.xmi when the app closes, recognized value: "true"
> "

**Comment by wolfgang-ch** [comment]
This is no issue, it's just an info
